{{cookiecutter.project_slug}}/.github/workflows/release.yaml

```diff
@@ -29,6 +33,29 @@ jobs:
       - name: Install deps
         run: poetry install
 
+      - name: Run semantic release (for tests)
+      if: ${{ github.event_name != 'workflow_dispatch' }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: make release-dry
+
+      - name: Release
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: make release-ci
+
+      - name: Generate documentation with changes from semantic-release
+      if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
you need to fix the indentation here as well
you can copy all the content of the file and check that here: https://www.yamllint.com/